CHANGELOG.md

```diff
@@ -26,7 +47,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Falsy OpenAPI `example` fields are validated correctly [#1418](https://github.com/stoplightio/spectral/pull/1418)
 - OpenAPI examples in examples is asked to be OpenAPI compliant [#1406](https://github.com/stoplightio/spectral/issues/1406)
 - Allow single trailing slash for OpenAPI 3 server url [#1427](https://github.com/stoplightio/spectral/pull/1427)
-- Missing non-existing enums in the OpenAPI default [#1429](https://github.com/stoplightio/spectral/pull/1429)
```

Comment:
Yup, this is on purpose.
Back in Jan 2021, while I cherry-picking changes, I realized that this particular commit cannot be easily integrated within the existing codebase, hence I had to pull it out.
I updated the changelog file on `master` branch respectively, but `develop` was kept untouched.
Now, I'm syncing the changelogs.